tools/image_processing/imagej2/imagej2_make_binary_jython_script.py

```diff
@@ -1,37 +1,36 @@
-import jython_utils
 import sys
+
 from ij import IJ
 
 # Fiji Jython interpreter implements Python 2.5 which does not
 # provide support for argparse.
-error_log = sys.argv[ -8 ]
-input = sys.argv[ -7 ]
-iterations = int( sys.argv[ -6 ] )
-count = int( sys.argv[ -5 ] )
-black_background = jython_utils.asbool( sys.argv[ -4 ] )
-pad_edges_when_eroding = jython_utils.asbool( sys.argv[ -3 ] )
-tmp_output_path = sys.argv[ -2 ]
-output_datatype = sys.argv[ -1 ]
+error_log = sys.argv[-8]
+input = sys.argv[-7]
+iterations = int(sys.argv[-6])
+count = int(sys.argv[-5])
+black_background = sys.argv[-4] == 'yes'
+pad_edges_when_eroding = sys.argv[-3] == 'yes'
+tmp_output_path = sys.argv[-2]
+output_datatype = sys.argv[-1]
 
 # Open the input image file.
-input_image_plus = IJ.openImage( input )
+input_image_plus = IJ.openImage(input)
 
 # Create a copy of the image.
 input_image_plus_copy = input_image_plus.duplicate()
 image_processor_copy = input_image_plus_copy.getProcessor()
 
-try:
-    # Set binary options.
-    options = jython_utils.get_binary_options( black_background=black_background,
-                                               iterations=iterations,
-                                               count=count,
-                                               pad_edges_when_eroding=pad_edges_when_eroding )
-    IJ.run( input_image_plus_copy, "Options...", options )
+# Set binary options.
+options = ['edm=Overwrite', 'iterations=%d' % iterations, 'count=%d' % count]
```

Comment:
have you ever thought to put this as well into the XML as config-file? This would at least save all the param passing

Reply:
I don't know if this call to IJ.run() can be done on the command line outside of Imagej2 scripting (https://imagej.net/Scripting).  If it is possible, I've not found examples of how to do it.  If we do find a way to do it, we can eliminate all of the Jython scripts for these tools in addition to all of the Python scripts that I am eliminating now.